Ginger/Ginger/ALM/Repository/OctaneRepository.cs

```diff
@@ -75,27 +86,117 @@ public override List<string> GetTestPlanExplorer(string path)
 
         public override IEnumerable<object> GetTestSetExplorer(string path)
         {
-            throw new NotImplementedException();
+            return octaneCore.GetTestSetExplorer(path); 
         }
 
         public override object GetTSRunStatus(object tsItem)
         {
-            throw new NotImplementedException();
+            return octaneCore.GetTSRunStatus(tsItem);
         }
 
         public override void ImportALMTests(string importDestinationFolderPath)
         {
-            throw new NotImplementedException();
+            Reporter.ToLog(eLogLevel.DEBUG, "Start importing from QC");
+            //set path to import to               
+            if (importDestinationFolderPath == "")
```

Comment:
Codacy found an issue: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=6317725)